sdk-lib/src/main/java/cash/z/ecc/android/sdk/db/entity/CompactBlockEntity.kt

```diff
@@ -5,7 +5,7 @@ import androidx.room.Entity
 
 @Entity(primaryKeys = ["height"], tableName = "compactblocks")
 data class CompactBlockEntity(
-    val height: Int,
+    val height: Long,
```

Comment:
Missed change from the BlockHeight API, so I fixed it.  SQLite is loose with types so no migration should be needed.